packages/clay-core/src/tree-view/TreeView.tsx

```diff
@@ -102,6 +102,7 @@ export function TreeView<T>({
 	onRenameItem,
 	onSelectionChange,
 	selectedKeys,
+	selectionHydrationMode = 'hydrate-first',
```

Comment:
Shouldn't this also be declared in [`ITreeViewProps`](https://github.com/liferay/clay/blob/a3db213cc1b92ca4b6b35bb0652169c16ea2418d/packages/clay-core/src/tree-view/TreeView.tsx#L19-L79) ?